pkg/cli/clusteroperator/restartkubelet/command.go

```diff
@@ -0,0 +1,92 @@
+package restartkubelet
+
+import (
+	"github.com/openshift/oc/pkg/cli/clusteroperator/pernodepod"
+
+	"github.com/spf13/cobra"
+	"k8s.io/cli-runtime/pkg/genericclioptions"
+	cmdutil "k8s.io/kubectl/pkg/cmd/util"
+	"k8s.io/kubectl/pkg/util/i18n"
+	"k8s.io/kubectl/pkg/util/templates"
+)
+
+var (
+	regenerateSignersLong = templates.LongDesc(`
+		Regenerate certificates provided by an OCP v4 cluster.
+		
+		This command does not wait for changes to be acknowledged by the cluster.
+		Some may take a very long time to roll out into a cluster, with different operators and operands involved for each.
+
+		Experimental: This command is under active development and may change without notice.
+	`)
+
+	regenerateSignersExample = templates.Examples(`
+		# Restart all the nodes,  10% at a time
+		oc adm clusteroperators restart-kubelet nodes --all
+
+		# Restart all the nodes,  20 nodes at a time
+		oc adm clusteroperators restart-kubelet nodes --all --parallelism=20
+
+		# Restart all the nodes,  15% at a time
+		oc adm clusteroperators restart-kubelet nodes --all --parallelism=15%
+
+		# Restart all the masters at the same time
+		oc adm clusteroperators restart-kubelet nodes -l node-role.kubernetes.io/master --parallelism=100%`)
```

Comment:
Hmm...ISTM we should be a bit more opinionated here.  First, the way the MCO works is to roll one controlplane and one worker simultaneously.  Instead of operating on nodes, we could actually operate on pools...

But even that aside I think we shouldn't document trying to roll the entire control plane simultaneously, right?  I mean, let's document doing that for the *workers* perhaps.

Reply:
> But even that aside I think we shouldn't document trying to roll the entire control plane simultaneously, right? I mean, let's document doing that for the workers perhaps.

A kubelet restart is non-disruptive.  It doesn't impact pod lifecycles. I think restarting all of them concurrently is acceptable.